core-common/src/main/java/org/uniprot/core/util/EnumDisplay.java

```diff
@@ -2,17 +2,26 @@
 
 import static org.uniprot.core.util.Utils.nullThrowIllegalArgument;
 
+import java.util.Arrays;
+
 import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
 public interface EnumDisplay {
 
+    /**
```

Comment:
some documentation is better than no docs
This method will return primary value of enum. It can be name or value or operation or etc